authentication/methods.mdx

```diff
@@ -176,12 +176,51 @@ If your Kubernetes environment has short-lived service account tokens, care will
 
 The client token produced can be used in subsequent API requests with the rest of the Flipt API to gain authenticated access.
 
+### Via the SDK
+
+Some of our SDKs support automatic authentication via the Kubernetes authentication method.
+These clients do not require you to have to manually invoke the verify service account.
+Instead, they do this operation for you, and they ensure that the retrieved client token from Flipt is automatically refreshed.
+
+The SDKs that currently support this include:
```

Comment:
on line `157` above we say:

> We're actively designing and developing client-side implementations to leverage this method seamlessly.
However, for now, the API must be managed manually by your implementation.

could we update or remove this as well?

Reply:
Good shout